docs/admin/authorization/rbac.md

```diff
@@ -693,11 +693,11 @@ subjects:
   name: user-1
 ```
 
-When bootstrapping the first roles and role bindings, it is necessary for the initial user to grant permissions they do not yet have.
-To bootstrap initial roles and role bindings:
-
-* Use a credential with the `system:masters` group, which is bound to the `cluster-admin` super-user role by the default bindings.
-* If your API server runs with the insecure port enabled (`--insecure-port`), you can also make API calls via that port, which does not enforce authentication or authorization.
+When bootstrapping the first roles and role bindings, it is necessary for the
+initial user to grant permissions they do not yet have.
+To bootstrap initial roles and role bindings, use a credential with the
+`system:masters` group, which is bound to the `cluster-admin` super-user role
+by the default bindings.
```

Comment:
Really? This seems like it runs counter to best practices in secure use of RBAC ...

Reply:
> Really? This seems like it runs counter to best practices in secure use of RBAC ...

That's how we (sig-auth) managed the initial bootstrapping, loopback connections, and "I accidentally deleted all permissions" problems.